### PR TITLE
Add documentation and screenshots for group projects

### PR DESCRIPTION
**Pull request by david-cako**
Adds necessary documentation and screenshots for group projects.  Changes instances of "Assignments" to "Projects".  Additional changes since screenshot below, instances of template changed to "project".

Screenshot_2020-10-13 Repl it - Project Workflow

### CODE REVIEW

File: sidebar.json [hunk 1]
```diff
@@ -81,8 +81,8 @@
         "slug": "Invitations"
       },
 						{
-        "name": "Assignment Workflow",
-        "slug": "Assignments"
+        "name": "Project Workflow",
+        "slug": "Projects"
```
Comment (sixhobbits) [inline sidebar.json:85]: image

Note that this change means that at least this would need to be updated too and maybe others.

Comment (david-cako) [reply sidebar.json:85]: very good point, ty

File: Teams/Projects.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+When you are done editing your project and it is ready to be published, click "Publish project" from the IDE.  
+
+<img style="max-width: 150px" src="/images/teamsForEducation/project-publish-button.png" />
+
+A modal appears, where you can toggle the project between "Published" and "Unpublished".
+
+**For individual projects**, students are able to fork the project and create their own copy. Their copy will be private so that only the student and team admins can see and edit their work.
+
+**For group projects**, you must assign students into submission groups to collaborate.  Click "Auto Assign" to automatically assign all students into equal groups.  You can add and remove new groups, drag and drop students between groups, and search for students.
```
Comment (jeremypress) [inline Teams/Projects.md:18]: do you think it's worth noting that groups cannot be removed after you save them? Something I noticed when messing around with it.

File: Teams/Projects.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+**For group projects**, you must assign students into submission groups to collaborate.  Click "Auto Assign" to automatically assign all students into equal groups.  You can add and remove new groups, drag and drop students between groups, and search for students. 
+
+Click "Save" to finalize any changes, or "Cancel" to discard changes.
+
+<img style="max-width: 600px" src="/images/teamsForEducation/project-publish.png" />
+
+## Submitting
+
+When students have completed their work on the project, click "Submit" in the IDE header, which will notify all team admins of their submission.
```
Comment (jeremypress) [inline Teams/Projects.md:26]: "they can click "submit"", i think our POV is of the instructor for these docs

File: Teams/Projects.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+<img style="max-width: 100px" src="/images/teamsForEducation/project-submit.png" />
+
+## Reviewing submissions
+
+Team admins are able to view project submissions from the team dashboard.  Click "View submissions" on the project.
```
Comment (jeremypress) [inline Teams/Projects.md:32]: let's lean in to the education workflow and just call them teachers or instructors

**Review by jeremypress — COMMENTED** [review]
looks good, couple of comments + what Gareth said about being consistent with "projects" vs. assignments. Also it looks like on all other pages we use a single space after a sentence, so let's do that here too.

**Comment by david-cako** [comment]
PR in to update link https://github.com/replit/repl-it-web/pull/7845

**Review by jeremypress — APPROVED** [review]
:pen: :framed_picture:

**Pull request merged**